plural/helm/console/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: console
 description: A chart for plural console
-appVersion: 0.5.5
+appVersion: 0.5.6

```

Comment:
this needs to be 0.5.4 since we have no docker tag at 0.5.5 or 0.5.6 (maps to gh releases)

Reply:
oh wait, I confused `appVersion` with `version` and bumped wrong one

Reply:
yeah

Reply:
done